src/test/java/codeu/controller/AdminPageServletTest.java

```diff
@@ -0,0 +1,57 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.UUID;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class AdminPageServletTest {
+
+  private ChatServlet adminServlet;
```

Comment:
should this be AdminServlet?

Reply:
yeah this was a commit i made before actually writing the test file! i meant to revert this commit too but i guess i didn’t o:

Reply:
yeah this was a commit i made before actually writing the test file! i meant to revert this commit too but i guess i didn’t o: